docs/disabled_features.md

```diff
@@ -5,18 +5,33 @@ sort_rank: 10
 
 # Disabled Features
 
-Here is a list of features that are disabled by default since they are breaking changes or are considered experimental.
```

Comment:
Can you please revert these changes? All the pages in the docs use "you". We should not change the pattern for just une file and changing all the doc is out of scope for this PR.

Reply:
No problem!  Reverted.